tests/decorators/test_cache_decorators.py

```diff
@@ -0,0 +1,97 @@
+from unittest import mock
```

Comment:
The existing test code for `cache` view decorators has been split out into its own file as per discussions in the [issue ticket](https://code.djangoproject.com/ticket/31949), and further tests will be added here in later commits. However, I'm not sure if`tests/decorators/test_cache_decorators.py` is a good naming convention / location for this test file?

Reply:
I changed to the `tests/decorators/test_cache.py` and also moved `DecoratorsTest.test_cache_page()` (see #16718).

Reply:
Merged in f2b97fbf065a823423e8ee2d1841b68ab22c604a. Please rebase from the `main` branch.